Source/Scene/SkyAtmosphere.js

```diff
@@ -81,6 +81,42 @@ define([
         this._spSkyFromSpace = undefined;
         this._spSkyFromAtmosphere = undefined;
 
+        this._spSkyFromSpaceColorCorrect = undefined;
+        this._spSkyFromAtmosphereColorCorrect = undefined;
+
+        /**
+         * Use hue/saturation/brightness shifts to postprocess the sky/atmosphere color.
+         * @type (Boolean)
+         * @default false
+         */
+        this.colorCorrect = false;
```

Comment:
Instead of this variable it's easier to just check in `update` that hue, saturation, or brightness are not 0.0.